### ISSUE TITLE
Tribute Page Project Instruction about using figure tag

### ISSUE THREAD

**Issue by navjotjsingh** [freeCodeCamp/freeCodeCamp#42392]
**Affected page**
Add a link to the coding challenge with the problem.
https://www.freecodecamp.org/learn/responsive-web-design/responsive-web-design-projects/build-a-tribute-page

User Story no. 3 mentions using a div tag with `id="img-div"` but your test and sample allow and uses a `<figure>` tag. The instruction needs to reflect this change.

**Comment by lasjorg** [comment]
I guess that's true. We (sort of silently) added the support for the figure element to allow for more semantic HTML. The problem is the awkwardly named id which we can't change without breaking backward compatibility with already finished projects.


> User Story #3: I should see a `div` or `figure` element with a corresponding id="img-div".

Asking the student to add a `figure` element with an id of `img-div` is awkward. But I guess there really isn't much we can do about it. I'm willing to allow the addition to the user story.

**Comment by raisedadead** [comment]
We have been discussing avoiding showing the source code of the Pens altogether - this came up during a contributors meeting. 

Yes - we should align the instructions as closely as possible with source/example templates but it often leads to users peeking under the hood. That's not necessarily a bad thing, but it often causes nuanced issues such as this where would be doing breaking non-compatible changes.

I think we should close this issue as a `wontfix` because of the legacy reasons, but I will seek an opinion from @moT01 first.

**Comment by moT01** [comment]
It looks like the test will allow any element as long as the `id` is correct:
```js
it(`I should see either a <figure> or <div> element with corresponding
id="img-div".`, function () {
  assert.isNotNull(document.getElementById('img-div'));
});
```

In the test suite, user story 3 says what @lasjorg mentioned:
![Screen Shot 2021-06-08 at 6 35 36 AM](https://user-images.githubusercontent.com/20648924/121178921-ab1b4100-c824-11eb-869f-2789e0587934.png)

It looks like user story 5 has a similar issue, the test suite doesn't match what's on the website. Again, this test just looks for the `id`. I do think something could be done here since the user stories don't align in the two spots. I kind of feel like we should just change user story 3 to:
`I should see a figure element with a corresponding id="img-div".`
and user story 5 to:
`Within the "img-div" element, I should see a <figcaption> element with a corresponding id="img-caption" that contains textual content describing the image shown in "img-div".`
Since those are probably the more "better practice" elements to use. This change would need to be done in both spots - and since the tests allow any element, no old projects would break. Although, like mentioned - the `id` names seem a little goofy if we do that, so the other option would be to just change what's on the website to match the test suite. I guess I'm kind of leaning towards doing that now since it would be easier. What do you think @navjotjsingh & @lasjorg - any other thoughts or ideas?

**Comment by lasjorg** [comment]
I updated this project 2 years ago and we haven't had any issues with the changes as far as I know.

When I did the update I improved the HTML semantics and initially was planning to update the user stories and tests. In order to support the legacy version of the project, I wanted to conditionally test for it. But at the time it was deemed simpler to just leave as many of the old tests as-is instead.

I don't think it would be too much work to update user stories and tests so we require semantic elements with new ids but still allow legacy projects to pass. The legacy project tests would be hidden from anyone currently doing the project and they would not know about the old requirements and user stories. But I'm not sure if it's really worth it as it will inevitably increase the complexity of the tests and their maintenance.

This is such a simple project and most people doing this project are super new to coding. Most people that do notice any discrepancies with our projects are not beginners.
